scripts/inventory/ovirt4.py

```diff
@@ -106,14 +107,24 @@ def create_connection():
     config_path = os.environ.get('OVIRT_INI_PATH', default_path)
 
     # Create parser and add ovirt section if it doesn't exist:
-    config = configparser.SafeConfigParser(
-        defaults={
-            'ovirt_url': os.environ.get('OVIRT_URL'),
-            'ovirt_username': os.environ.get('OVIRT_USERNAME'),
-            'ovirt_password': os.environ.get('OVIRT_PASSWORD'),
-            'ovirt_ca_file': os.environ.get('OVIRT_CAFILE', ''),
-        }
-    )
+    if PY2:
+        config = configparser.SafeConfigParser(
+            defaults={
+                'ovirt_url': os.environ.get('OVIRT_URL'),
+                'ovirt_username': os.environ.get('OVIRT_USERNAME'),
+                'ovirt_password': os.environ.get('OVIRT_PASSWORD'),
+                'ovirt_ca_file': os.environ.get('OVIRT_CAFILE', ''),
+            }
```

Comment:
Why not pass `allow_no_value=True` here as well?

Reply:
it is working without it, we can add it but it is not necessaryry

Reply:
So the change is only needed for Python 3?

Reply:
yes